cpp/tools/ZeroC.Ice.Slice.Tools.Cpp/BUILDING.md

```diff
@@ -0,0 +1,25 @@
+# Building Slice Tools for Ice and C++
```

Comment:
If we settle on this name - Slice Tools for Ice and C++ - I would add "the" to this title:

Building the Slice Tools for Ice and C++